lib/sequel/adapters/shared/mysql.rb

```diff
@@ -577,8 +577,11 @@ def schema_parse_table(table_name, opts)
             row[:generated] = !!(extra.to_s =~ /VIRTUAL|STORED|PERSISTENT/i)
           end
           row[:allow_null] = row.delete(:Null) == 'YES'
+          row[:collation] = row.delete(:Collation)
```

Comment:
Collation should not be included, nor privileges.  This PR is only for comment support.